### DISCUSSION TITLE
Use @primevue/forms with CDN

### DISCUSSION THREAD

**Discussion by Remo** [primefaces/community#3130] [PrimeVue]
We've got a site that is set up like this https://primevue.org/cdn/

It all works very well, but now that we wanted to use the forms component (https://primevue.org/forms), we seem to have hit a dead end. Nothing in prime.min.js is related to forms and we cannot load it using unpkg. Any idea?

**Comment by 1Map** [comment]
True, I also require the umd from the cdn.....

For example:

https://cdn.jsdelivr.net/npm/primevue@4.2.5/umd/primevue.min.js

Currently I only see:

https://unpkg.com/@primevue/forms@4.2.5/index.mjs

For Forms.

But I require umd for Forms

@tugcekucukoglu Any idea on this?
